.travis.yml

```diff
@@ -5,7 +5,8 @@ android:
   components:
     - build-tools-21.1.2
     - android-21
+    - extra
```

Comment:
couldn't find support library without it
https://travis-ci.org/venmo/cursor-utils/builds/46250306